jsx/StaticDataTable.js

```diff
@@ -8,7 +8,13 @@ StaticDataTable = React.createClass({
         // func(ColumnName, CellData, EntireRowData)
         getFormattedCell: React.PropTypes.func
     },
-
+    componentDidMount: function() {
+        if(this.props.freezeColumn) {
```

Comment:
can you add an if (jQuery.fn.DynamicTable) {} around this whole if/else block so that it the whole table doesn't crash if, for some reason, the dynamicTable plugin isn't loaded?